cmd/instance/instance_vnc.go

```diff
@@ -75,27 +78,27 @@ Duration follows Go's duration format (e.g. "30m", "1h", "24h")`,
 
 		err = waitEndpointReady(vnc.URI)
 		if err != nil {
-			utility.Error("VNC Console URL is not reachable: %s", err)
+			utility.Error("Console URL is not reachable: %s", err)
 			os.Exit(1)
 		}
 
-		utility.Info("Opening VNC in your default browser...")
+		utility.Info("Opening the Console in your default browser...")
```

Comment:
Would change "Opening the console" to maintain capitalization consistency

Reply:
That works, but given it's a full sentence my brain would wish an article (a/the) in the sentence.